diff --git a/testing/runtests.jl b/testing/runtests.jl
index 2a80016..bbfff58 100644
--- a/testing/runtests.jl
+++ b/testing/runtests.jl
@@ -67,11 +67,10 @@ end
 
 println("Running tests...")
 if !isinteractive()
-    packmol = joinpath(@__DIR__,"..","packmol")
     for input_test in ARGS
         print(" Running test $input_test ...")
         log = IOBuffer()
-        run(pipeline(`$packmol`; stdin=input_test, stdout=log))
+        run(pipeline(`packmol`; stdin=input_test, stdout=log))
         if occursin("Success!", String(take!(log)))
             check_mind(input_test)
         else
diff --git a/testing/test_failed.sh b/testing/test_failed.sh
index 9cf19f9..ae6471a 100755
--- a/testing/test_failed.sh
+++ b/testing/test_failed.sh
@@ -12,7 +12,7 @@ if ! [ -f $1 ]; then
     echo "Error: input file not found: $1"
     exit 1
 fi
-../packmol < $1 > $2
+packmol < $1 > $2
 if ! [ -f $2 ]; then
     echo "Error: Log file not generated: $2"
     exit 1
