diff --git a/testing/test_failed.sh b/testing/test_failed.sh
--- a/testing/test_failed.sh
+++ b/testing/test_failed.sh
@@ -13,7 +13,7 @@
     echo "Error: input file not found: $1"
     exit 1
 fi
-../packmol < $1 > $2
+packmol < $1 > $2
 if ! [ -f $2 ]; then
     echo "Error: Log file not generated: $2"
     exit 1
diff --git a/testing/runtests.jl b/testing/runtests.jl
--- a/testing/runtests.jl
+++ b/testing/runtests.jl
@@ -85,7 +85,7 @@
         popat!(ARGS, packmol_arg + 1)
         popat!(ARGS, packmol_arg)
     else
-        packmol = joinpath(@__DIR__,"..","packmol")
+        packmol = "packmol"
     end
     println(" Packmol path: $packmol")
     for input_test in ARGS
